app/src/main/res/values/strings.xml

```diff
@@ -59,22 +59,22 @@
     <string name="importOptionFilesystemTitle">Import from filesystem</string>
     <string name="importOptionFilesystemExplanation">Choose a specific file from the filesystem.</string>
     <string name="importOptionFilesystemButton">From filesystem</string>
-    <string name="importOptionApplicationTitle">Use external app</string>
+    <string name="importOptionApplicationTitle">Use another app</string>
     <string name="importOptionApplicationExplanation">Use any app or your favorite file manager to open a file.</string>
-    <string name="importOptionApplicationButton">Use external app</string>
+    <string name="importOptionApplicationButton">Use another app</string>
 
     <string name="about">About</string>
-    <string name="app_copyright_fmt" translatable="false" tools:ignore="PluralsCandidate">Copyright 2019–<xliff:g>%d</xliff:g> Sylvia van Os.</string>
-    <string name="app_copyright_old">Based on Loyalty Card Keychain, copyright 2016–2020 Branden Archer.</string>
-    <string name="app_license">Copylefted libre software, licensed GPLv3+.</string>
+    <string name="app_copyright_fmt" tools:ignore="PluralsCandidate">Copyright © 2019–<xliff:g>%d</xliff:g> Sylvia van Os.</string>
+    <string name="app_copyright_old">Based on Loyalty Card Keychain\ncopyright © 2016–2020 Branden Archer.</string>
+    <string name="app_license">🄯 Copylefted libre software, licensed GPLv3+.</string>
     <string name="about_title_fmt">About <xliff:g id="app_name">%s</xliff:g></string>
     <string name="debug_version_fmt">Version: <xliff:g id="version">%s</xliff:g></string>
     <string name="app_revision_fmt">Revision Info: <xliff:g id="app_revision_url">%s</xliff:g></string>
-    <string name="app_libraries"><xliff:g id="app_name">%s</xliff:g> uses the following third-party libraries: <xliff:g id="app_libraries_list">%s</xliff:g></string>
-    <string name="app_resources"><xliff:g id="app_name">%s</xliff:g> uses the following third-party resources: <xliff:g id="app_resources_list">%s</xliff:g></string>
+    <string name="app_libraries">Libre software libraries: <xliff:g id="app_libraries_list">%s</xliff:g></string>
+    <string name="app_resources">Libre resources: <xliff:g id="app_resources_list">%s</xliff:g></string>
 
-    <string name="selectBarcodeTitle">Select Barcode</string>
-    <string name="enterBarcodeInstructions">Enter the card ID then select the image which represents the barcode you want to use, or select &#8220;This card has no barcode&#8221; to not use a barcode.</string>
+    <string name="selectBarcodeTitle">Scan instead</string>
```

Comment:
Found this in manual scan, is it used elsewhere? Now it only works there.

Reply:
Where did you find it? Because from looking at the code the only place is as the title of the SelectBarcodeActivity:

![image](https://user-images.githubusercontent.com/1885159/113523556-61a24180-95a8-11eb-82cc-9e8ad6d183c4.png)

Changing this to "Scan instead" is definitely 100% incorrect.

Reply:
Yes, and when clicked there, the user is taken back to the default scan.
Thus "Scan instead" as opposed to ~"enter manually".

Reply:
I'm not accepting a change to turn this to Scan Instead, simply because that is not as per Android guidelines and how other apps work. Deviating from other apps will cause confusion. Besides, your concern should have been addressed by my change for #64 already.

Reply:
Android guidelines do follow my idea of usability eventually, but I guess we get there in the end.